src/Aspire.Hosting.Azure.AIFoundry/api/Aspire.Hosting.Azure.AIFoundry.cs

```diff
@@ -25,6 +27,151 @@ public static ApplicationModel.IResourceBuilder<T> WithRoleAssignments<T>(this A
+            public static readonly AIFoundryModel EmbedV40;
+        }
+
+        public static partial class Core42
+        {
+            public static readonly AIFoundryModel Jais30bChat;
+        }
+
+        public static partial class DeepSeek
+        {
+            public static readonly AIFoundryModel DeepseekR1;
+            public static readonly AIFoundryModel DeepseekR10528;
+            public static readonly AIFoundryModel DeepseekV3;
+            public static readonly AIFoundryModel DeepseekV30324;
+        }
+
+        public static partial class Meta
+        {
+            public static readonly AIFoundryModel Llama3211BVisionInstruct;
+            public static readonly AIFoundryModel Llama3290BVisionInstruct;
+            public static readonly AIFoundryModel Llama3370BInstruct;
+            public static readonly AIFoundryModel Llama4Maverick17B128EInstructFp8;
+            public static readonly AIFoundryModel Llama4Scout17B16EInstruct;
+            public static readonly AIFoundryModel MetaLlama31405BInstruct;
+            public static readonly AIFoundryModel MetaLlama3170BInstruct;
+            public static readonly AIFoundryModel MetaLlama318BInstruct;
+            public static readonly AIFoundryModel MetaLlama370BInstruct;
+            public static readonly AIFoundryModel MetaLlama38BInstruct;
+        }
+
+        public static partial class Microsoft
+        {
+            public static readonly AIFoundryModel AzureAiContentSafety;
+            public static readonly AIFoundryModel AzureAiContentUnderstanding;
+            public static readonly AIFoundryModel AzureAiDocumentIntelligence;
+            public static readonly AIFoundryModel AzureAiLanguage;
+            public static readonly AIFoundryModel AzureAiSpeech;
+            public static readonly AIFoundryModel AzureAiTranslator;
+            public static readonly AIFoundryModel AzureAiVision;
+            public static readonly AIFoundryModel MaiDsR1;
+            public static readonly AIFoundryModel ModelRouter;
+            public static readonly AIFoundryModel Phi35MiniInstruct;
+            public static readonly AIFoundryModel Phi35MoeInstruct;
+            public static readonly AIFoundryModel Phi35VisionInstruct;
+            public static readonly AIFoundryModel Phi3Medium128kInstruct;
+            public static readonly AIFoundryModel Phi3Medium4kInstruct;
+            public static readonly AIFoundryModel Phi3Mini128kInstruct;
+            public static readonly AIFoundryModel Phi3Mini4kInstruct;
+            public static readonly AIFoundryModel Phi3Small128kInstruct;
+            public static readonly AIFoundryModel Phi3Small8kInstruct;
+            public static readonly AIFoundryModel Phi4;
+            public static readonly AIFoundryModel Phi4MiniInstruct;
+            public static readonly AIFoundryModel Phi4MiniReasoning;
+            public static readonly AIFoundryModel Phi4MultimodalInstruct;
```

Comment:
Pretty sure these are broken. We should try to check in the JSON response file as part of 9.5 so we can properly handle this.

Reply:
cc @sebastienros

Reply:
Fixed in https://github.com/dotnet/aspire/pull/11578